src/interface/src/app/treatments/treatment-project-area/treatment-project-area.component.ts

```diff
@@ -51,4 +67,86 @@ export class TreatmentProjectAreaComponent implements OnDestroy {
     this.selectedStandsState.clearStands();
     this.treatmentsState.setShowApplyTreatmentsDialog(false);
   }
+
+  treatmentPlanId: number = this.route.snapshot.data['treatmentId'];
```

Comment:
ah- forgot to mention this but you should use the treatmentState instead for this? like we are already handling this stuff there. Not a blocker but that way we dont couple this component with route stuff.